benches/linkbench/app.lua

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env tarantool
+
+local path = require('fio').dirname(arg[0])
+package.path = path.."/?.lua;"..package.path
+package.cpath = path.."/?.so;"..package.cpath
+
+require('console').listen('unix/:./tarantool.sock')
+require('gperftools').cpu.start('tarantool.prof')
```

Comment:
Please, keep the relevant issue in the mind: https://github.com/tarantool/gperftools/issues/5

If you don't plan to use gperftools now, I suggest to don't call it.

Reply:
Unfortunately most of this code is being copied from according benchs/* code as is. It's just how it used to be and I think we'd better keep everything as it used to be. I would really love to rewrite some of the benchmarks in cleaner code/remove some unused code, but for now I suppose it is only needed when benchmark code is **so** bad, it would even fail to run sometimes

Reply:
Just observed: now there are bench/ and benches/ directories. I didn't get your idea here.

Reply:
The scripts contained in benchs/ are the old scripts that used to be run inside docker and had some complications to be run not in closed environments. Patching those scripts for both docker usage and local usage seemed to be very complicating task. So I decided to copy-paste them and modify the new ones. Most of the changes were applied to bash scripts, not the lua code for tarantool. So this is the lua code that is being run now inside docker in gitlab ci. I would really like to change this code in many of benchmarks, but [this](https://github.com/tarantool/bench-run/issues/17) is the issue to resolved in the future.

Reply:
Can we fork a branch for the development purposes? I see, you proposed the same in https://github.com/tarantool/bench-run/pull/14#discussion_r547360618. It looks worthful if you plan to use the `ttbench` script in tarantool's CI in a foreseeable future (within 2 months).

The only thing I want regarding the branch: it should be rebaseable on top of `master` and should land to `master` when we'll ready to use it in tarantool CI.

Reply:
But anyway, please, ensure that the changes that you made around the lua scripts may be observed in the diff. Now it looks like you wrote those scripts from scratch.